### ISSUE TITLE
[BLOCKING] Build cost more than ONE DAY and is still building

### ISSUE THREAD

**Issue by colingonglin** [GitbookIO/feedback#548]
## Category

BUG

## What is the current behavior?

I have 2 commits building for a long time. The longer one has built for more than 30 hours.

## What is the desired behavior?

Normally it should cost about 20 mins.

## Additional information

Build No. #8855ae8aae

**Comment by rcasalla** [comment]
I´m having the same problem. The builts are taking too much time ( #0d59befadd )

**Comment by jeanregisser** [comment]
Same here, my builds take ~3 hours and fail with the following output:

```
HTTP ERROR 504: ""
```

This is for https://www.gitbook.com/book/jeanregisser/mediaremotetv-protocol

**Comment by AaronO** [comment]
@colingonglin @rcasalla Builds should be fixed since about 3 days ago for good, I deployed 3 different improvements to avoid build delays happening in the future

**Comment by AaronO** [comment]
@jeanregisser What you're encountering is a different issue, and seems to a bug with `include-codeblock`.

I pulled your project locally and tried to build it, but it after ~20mn. I tried disabling the `include-codeblock` plugin and it build in under 2s.

It seems like `include-codeblock` is stalling for some reason, it's a 3rd party plugin so I'm unsure why. Would need to do further digging / debugging on that specific plugin.

Maybe you try to import a file that doesn't exist / wrong path and it doesn't handle that well.

Closing this issue, since delayed builds are not an issue and @jeanregisser's issue is with https://github.com/azu/gitbook-plugin-include-codeblock
